Silicon/CommonSocPkg/Library/PchSpiLib/PchSpiLib.c

```diff
@@ -83,6 +94,13 @@ EnableBiosWriteProtect (
   IN  UINTN         PchSpiBase
   )
 {
+  UINT8   BiosCtrl;
+
+  // Set the BIOS Lock Enable and EISS bits
+  BiosCtrl = MmioRead8 (PchSpiBase + R_SPI_BCR);
+  if (((BiosCtrl & B_SPI_BCR_SMM_BWP) != 0) && ((BiosCtrl & B_SPI_BCR_BLE) == 0)) {
+    MmioOr8(PchSpiBase + R_SPI_BCR, (UINT8)(B_SPI_BCR_BLE|B_SPI_BCR_SMM_BWP));

```

Comment:
Need to remove BLE. After setting BLE, no more SPI writing available.

Reply:
That is right.  BLE should be set at the very end.

Reply:
Thanks @myguitar and @mauricema for the review, I've removed the BLE.